2023.7.3

frenck opened the pull request:
- Use device class naming for SimpliSafe ([@joostlek] - [#96093]) ([simplisafe docs])
- Always try PullPoint with ONVIF ([@bdraco] - [#96377]) ([onvif docs])
- Bump vallox-websocket-api to 3.3.0 ([@slovdahl] - [#96493]) ([vallox docs]) (dependency)
- Fix Vallox fan entity naming ([@slovdahl] - [#96494]) ([vallox docs])
- Bump devolo_plc_api to 1.3.2 ([@Shutgun] - [#96499]) ([devolo_home_network docs]) (dependency)
- Support MyStrom switch 120 ([@joostlek] - [#96535]) ([mystrom docs])
- Defer SSDP UPNP server start until the started event ([@bdraco] - [#96555]) ([ssdp docs])
- Handle empty strings for ESPHome UOMs ([@bdraco] - [#96556]) ([esphome docs])
- Bump onvif-zeep-async to 3.1.12 ([@bdraco] - [#96560]) ([onvif docs]) (dependency)
- Fix daikin missing key after migration ([@mover85] - [#96575]) ([daikin docs])
- Ensure androidtv_remote does not block startup of HA ([@quthla] - [#96582]) ([androidtv_remote docs])
- Bump pyrainbird to 2.1.1 ([@allenporter] - [#96601]) ([rainbird docs]) (dependency)
- Bump pyrainbird to 3.0.0 ([@allenporter] - [#96610]) ([rainbird docs]) (dependency)
- Fix SmartThings Cover Set Position (for window shades) ([@andrewsayre] - [#96612]) ([smartthings docs]) (dependency)
- Improve imap error handling for config entry ([@jbouwh] - [#96724]) ([imap docs])
- Use explicit device name for Stookalert ([@joostlek] - [#96755]) ([stookalert docs])
- Prevent otbr creating multiple config entries ([@emontnemery] - [#96783]) ([otbr docs])
- Fix check for HA Yellow radio in otbr config flow ([@emontnemery] - [#96789]) ([otbr docs])
- Update PyYAML to 6.0.1 ([@cdce8p] - [#96800]) (dependency)
- Update pycocotools to 2.0.6 ([@cdce8p] - [#96831]) ([tensorflow docs]) (dependency)
- Add username to Reauth flow in Honeywell ([@mkmer] - [#96850]) ([honeywell docs])
- bump python-Roborock to 0.30.1 ([@Lash-L] - [#96877]) ([roborock docs]) (dependency)
- Avoid infinite loop on corrupt stream recording ([@uvjustin] - [#96881]) ([stream docs])
- Bump AIOSomecomfort to 0.0.15 in Honeywell ([@mkmer] - [#96904]) ([honeywell docs]) (dependency)
- Fix timer switch in Sensibo ([@gjohansson-ST] - [#96911]) ([sensibo docs])
- Disable wheels building for pycocotools ([@frenck] - [#96937])
- Fix esphome entity ids in test ([@bdraco] - [#96965]) ([esphome docs])
- Add custom bypass night arming to SIA alarm codes ([@etnoy] - [#95736]) ([sia docs])
- Upgrade pymazda to 0.3.10 ([@bdr99] - [#96954]) ([mazda docs]) (dependency)
- Fix OTBR tests on RC ([@frenck] - [#96966]) ([otbr docs])
- Update aiohttp to 3.8.5 ([@frenck] - [#96945]) (dependency)
- Remove stateclass from Systemmonitor process sensor ([@joostlek] - [#96973]) ([systemmonitor docs])
- Fix task leak on config entry unload/retry ([@bdraco] - [#96981])
- Bump env_canada to v0.5.36 ([@michaeldavie] - [#96987]) ([environment_canada docs])
- Update pyfibaro dependency ([@rappenze] - [#97004]) ([fibaro docs]) (dependency)

[#95736]: https://github.com/home-assistant/core/pull/95736
[#95908]: https://github.com/home-assistant/core/pull/95908
[#96006]: https://github.com/home-assistant/core/pull/96006
[#96093]: https://github.com/home-assistant/core/pull/96093
[#96377]: https://github.com/home-assistant/core/pull/96377
[#96487]: https://github.com/home-assistant/core/pull/96487
[#96493]: https://github.com/home-assistant/core/pull/96493
[#96494]: https://github.com/home-assistant/core/pull/96494
[#96499]: https://github.com/home-assistant/core/pull/96499
[#96535]: https://github.com/home-assistant/core/pull/96535
[#96555]: https://github.com/home-assistant/core/pull/96555
[#96556]: https://github.com/home-assistant/core/pull/96556
[#96560]: https://github.com/home-assistant/core/pull/96560
[#96575]: https://github.com/home-assistant/core/pull/96575
[#96582]: https://github.com/home-assistant/core/pull/96582
[#96601]: https://github.com/home-assistant/core/pull/96601
[#96610]: https://github.com/home-assistant/core/pull/96610
[#96612]: https://github.com/home-assistant/core/pull/96612
[#96724]: https://github.com/home-assistant/core/pull/96724
[#96755]: https://github.com/home-assistant/core/pull/96755
[#96783]: https://github.com/home-assistant/core/pull/96783
[#96789]: https://github.com/home-assistant/core/pull/96789
[#96800]: https://github.com/home-assistant/core/pull/96800
[#96831]: https://github.com/home-assistant/core/pull/96831
[#96850]: https://github.com/home-assistant/core/pull/96850
[#96877]: https://github.com/home-assistant/core/pull/96877
[#96881]: https://github.com/home-assistant/core/pull/96881
[#96904]: https://github.com/home-assistant/core/pull/96904
[#96911]: https://github.com/home-assistant/core/pull/96911
[#96937]: https://github.com/home-assistant/core/pull/96937
[#96945]: https://github.com/home-assistant/core/pull/96945
[#96954]: https://github.com/home-assistant/core/pull/96954
[#96965]: https://github.com/home-assistant/core/pull/96965
[#96966]: https://github.com/home-assistant/core/pull/96966
[#96973]: https://github.com/home-assistant/core/pull/96973
[#96981]: https://github.com/home-assistant/core/pull/96981
[#96987]: https://github.com/home-assistant/core/pull/96987
[#97004]: https://github.com/home-assistant/core/pull/97004
[@Lash-L]: https://github.com/Lash-L
[@Shutgun]: https://github.com/Shutgun
[@allenporter]: https://github.com/allenporter
[@andrewsayre]: https://github.com/andrewsayre
[@balloob]: https://github.com/balloob
[@bdr99]: https://github.com/bdr99
[@bdraco]: https://github.com/bdraco
[@cdce8p]: https://github.com/cdce8p
[@emontnemery]: https://github.com/emontnemery
[@etnoy]: https://github.com/etnoy
[@frenck]: https://github.com/frenck
[@gjohansson-ST]: https://github.com/gjohansson-ST
[@jbouwh]: https://github.com/jbouwh
[@joostlek]: https://github.com/joostlek
[@michaeldavie]: https://github.com/michaeldavie
[@mkmer]: https://github.com/mkmer
[@mover85]: https://github.com/mover85
[@quthla]: https://github.com/quthla
[@rappenze]: https://github.com/rappenze
[@slovdahl]: https://github.com/slovdahl
[@uvjustin]: https://github.com/uvjustin
[abode docs]: https://www.home-assistant.io/integrations/abode/
[accuweather docs]: https://www.home-assistant.io/integrations/accuweather/
[acmeda docs]: https://www.home-assistant.io/integrations/acmeda/
[androidtv_remote docs]: https://www.home-assistant.io/integrations/androidtv_remote/
[daikin docs]: https://www.home-assistant.io/integrations/daikin/
[devolo_home_network docs]: https://www.home-assistant.io/integrations/devolo_home_network/
[environment_canada docs]: https://www.home-assistant.io/integrations/environment_canada/
[esphome docs]: https://www.home-assistant.io/integrations/esphome/
[fibaro docs]: https://www.home-assistant.io/integrations/fibaro/
[honeywell docs]: https://www.home-assistant.io/integrations/honeywell/
[imap docs]: https://www.home-assistant.io/integrations/imap/
[mazda docs]: https://www.home-assistant.io/integrations/mazda/
[mystrom docs]: https://www.home-assistant.io/integrations/mystrom/
[onvif docs]: https://www.home-assistant.io/integrations/onvif/
[otbr docs]: https://www.home-assistant.io/integrations/otbr/
[rainbird docs]: https://www.home-assistant.io/integrations/rainbird/
[roborock docs]: https://www.home-assistant.io/integrations/roborock/
[sensibo docs]: https://www.home-assistant.io/integrations/sensibo/
[sia docs]: https://www.home-assistant.io/integrations/sia/
[simplisafe docs]: https://www.home-assistant.io/integrations/simplisafe/
[smartthings docs]: https://www.home-assistant.io/integrations/smartthings/
[ssdp docs]: https://www.home-assistant.io/integrations/ssdp/
[stookalert docs]: https://www.home-assistant.io/integrations/stookalert/
[stream docs]: https://www.home-assistant.io/integrations/stream/
[systemmonitor docs]: https://www.home-assistant.io/integrations/systemmonitor/
[tensorflow docs]: https://www.home-assistant.io/integrations/tensorflow/
[vallox docs]: https://www.home-assistant.io/integrations/vallox/

Docs PR: <https://github.com/home-assistant/home-assistant.io/pull/28258>

joostlek committed:
Use device class naming for SimpliSafe (#96093)

slovdahl committed:
Fix Vallox fan entity naming (#96494)

slovdahl committed:
Bump vallox-websocket-api to 3.3.0 (#96493)

Shutgun committed:
Bump devolo_plc_api to 1.3.2 (#96499)

joostlek committed:
Support MyStrom switch 120 (#96535)

bdraco committed:
Handle empty strings for ESPHome UOMs (#96556)

bdraco committed:
Defer SSDP UPNP server start until the started event (#96555)

bdraco committed:
Bump onvif-zeep-async to 3.1.12 (#96560)

bdraco committed:
Always try PullPoint with ONVIF (#96377)

mover85 committed:
Fix daikin missing key after migration (#96575)

Fix daikin migration

allenporter committed:
Bump pyrainbird to 2.1.1 (#96601)

allenporter committed:
Bump pyrainbird to 3.0.0 (#96610)

jbouwh committed:
Improve imap error handling for config entry (#96724)

* Improve error handling config entry

* Removed CancelledError

* Add cleanup

* Do not call protected async_set_state()

joostlek committed:
Use explicit device name for Stookalert (#96755)

emontnemery committed:
Prevent otbr creating multiple config entries (#96783)

emontnemery committed:
Fix check for HA Yellow radio in otbr config flow (#96789)

cdce8p committed:
Update PyYAML to 6.0.1 (#96800)

cdce8p committed:
Update pycocotools to 2.0.6 (#96831)

andrewsayre committed:
Fix SmartThings Cover Set Position (for window shades) (#96612)

* Update smartthings dependencies

* Update cover to support window_shade_level

Lash-L committed:
bump python-Roborock to 0.30.1 (#96877)

bump to 0.30.1

uvjustin committed:
Avoid infinite loop on corrupt stream recording (#96881)

* Avoid infinite loop on corrupt stream recording

* Update tests

mkmer committed:
Add username to Reauth flow in Honeywell (#96850)

* pre-populate username/password on reauth

* Update homeassistant/components/honeywell/config_flow.py

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

* Use add_suggested_value_to_schema

* Optimize code

---------

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

mkmer committed:
Bump AIOSomecomfort to 0.0.15 in Honeywell (#96904)

quthla committed:
Ensure androidtv_remote does not block startup of HA (#96582)

* Ensure androidtv_remote does not block startup of HA

* Fix lint

* Use asyncio.wait_for

* Update homeassistant/components/androidtv_remote/__init__.py

Co-authored-by: Erik Montnemery <erik@montnemery.com>

* Update homeassistant/components/androidtv_remote/__init__.py

Co-authored-by: Erik Montnemery <erik@montnemery.com>

* Fix lint

* Lint

* Update __init__.py

---------

Co-authored-by: Erik Montnemery <erik@montnemery.com>

gjohansson-ST committed:
Fix timer switch in Sensibo (#96911)

frenck committed:
Bumped version to 2023.7.3

frenck commented:
blocked by #96831

frenck committed:
Disable wheels building for pycocotools (#96937)

etnoy committed:
Add custom bypass night arming to SIA alarm codes (#95736)

* Add SIA codes for night arming with custom bypass

* Set night custom bypass to ARMED_CUSTOM_BYPASS

dshokouhi commented:
Shoudnt this also include a fronted fix for the history issue?

frenck committed:
Update aiohttp to 3.8.5 (#96945)

bdr99 commented:
If it's not too late, can https://github.com/home-assistant/core/pull/96954 be added to this patch release please?

bdr99 committed:
Upgrade pymazda to 0.3.10 (#96954)

frenck commented:
> If it's not too late, can #96954 be added to this patch release please?

Tagged it, need to restart CI anyways, so will pick it.

bdr99 commented:
@frenck Thanks!

bdraco committed:
Fix esphome entity ids in test (#96965)

frenck committed:
Fix OTBR tests on RC (#96966)

frenck commented:
Consider the hour, and still failing CI. I'm going to put this into draft and see how tonights nightlight is going to run. Mostly to maybe include the aiohttp upgrade.

joostlek commented:
If it's going to take longer, is there a way we can put that frontend fix in for freezing dashboards?

frenck commented:
> If it's going to take longer, is there a way we can put that frontend fix in for freezing dashboards?

We cannot easily ship a frontend at this point, mostly due to a lack of resources. The next frontend will be in the 2023.8.0 beta.

bdraco commented:
Just found a bad memory leak https://github.com/home-assistant/core/issues/96980

bdraco commented:
working on a fix in https://github.com/home-assistant/core/pull/96981

bdraco committed:
Fix task leak on config entry unload/retry (#96981)

Since the task was added to self._tasks without a `task.add_done_callback(self._tasks.remove)`
each unload/retry would leak a new set of tasks

michaeldavie committed:
Bump env_canada to v0.5.36 (#96987)

joostlek committed:
Remove stateclass from Systemmonitor process sensor (#96973)

Remove stateclass

rappenze committed:
Update pyfibaro dependency (#97004)

cybernard commented:
python-roborock has been updated again to 30.2 please include in the next update to homeassistant
